src/isValidRequest.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable max-len */
+
+function isValidReq(text, searchProperty, searchParam) {
+  let error = false;
+  let message = '';
+  const cases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
```

Comment:
Move the 'cases' array outside of the function to avoid recreating it every time the function is called.